ceph-mixin/alerts/CephClusterAlerts.yml

```diff
@@ -1,7 +1,7 @@
 groups:
 - name: CephClusterAlerts
   rules:
-  - alert: CephUnhealthyWarning
+  - alert: CephUnhealthyWarining
```

Comment:
Introduces typo?

Reply:
Ah good catches, I think the mixin version used in the integration must have been out of date and I used that version when updating. Will go back and double check this one!